ansible/tasks/test-image.yml

```diff
@@ -52,11 +52,11 @@
   when: ebssurrogate_mode
 
 - name: Drop pgTap extension
-  shell: /usr/lib/postgresql/bin/psql -U postgres -h localhost -d postgres -c "DROP extension if exists pgtap";
+  shell: /usr/lib/postgresql/bin/psql -U supabase_admin -h localhost -d postgres -c "DROP extension if exists pgtap";
```

Comment:
probably better to wrap creation of pgtap extension in the txn instead  https://github.com/supabase/postgres/blob/develop/migrations/tests/test.sql#L4 

Then we can remove this step.